api/src/test/java/uk/gov/hmcts/payment/api/componenttests/AccountServiceTest.java

```diff
@@ -41,7 +41,7 @@ public class AccountServiceTest {
     @Test
     public void retrieveExistingAccountReturnsAccountDto() throws Exception {
         String pbaCode = "PBA1234";
-        FieldSetter.setField(accountServiceImpl, accountServiceImpl.getClass().getDeclaredField("baseUrl"), baseUrl);
```

Comment:
Internal method that's no longer accessible in later versions of Mockito - replaced with an equivalent.